curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/666ab82c874add3c95145213.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 666ab82c874add3c95145213
+title: Step 7
+challengeType: 20
+dashedName: step-7
+---
+
+# --description--
+
+One last important method, for now, is the `pop()` method. `pop()` can be used to remove any one element from a list. By default, it removes the last element of the list, but you can add an argument and it will remove the element at the given index.
```

Comment:
```suggestion
One last important method, for now, is the `pop()` method. `pop()` can be used to remove an element from a list. By default, it removes the last element of the list, but you can pass an argument to the method and it will remove the element at the given index.
```